style.css

```diff
@@ -82,3 +82,196 @@ ul {
   text-align: center;
   margin: 10px;
 }
+
+.page1 {
+  margin: 100px 40px;
+  box-sizing: border-box;
+  border: 1px solid #fff;
+  border-radius: 16px;
+  height: 552px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  padding: 16px;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.image1 {
+  width: 100%;
+  height: auto;
+}
+
+.list {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 20%;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.text1-1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+}
+
+.dot {
+  height: 8px;
+  width: 8px;
+  border-radius: 50%;
+  background-color: #7a869a;
+}
+
+.can {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.Bac {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.par {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 10px;
+}
+
+.see {
+  margin-top: 14px;
+}
+
+.button {
+  border: none;
+  cursor: pointer;
+  padding: 4px 12px;
+  margin-top: 12px;
+  width: auto;
+  height: auto;
+  background: #ebebff;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.btngr {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  width: auto;
+  height: auto;
+  padding: 5px 5px;
+  cursor: pointer;
+  border-color: #396df2;
+  border-radius: 8px;
+  background-color: #fff;
+}
+
+#home1 {
+  background-color: #fff;
+  border-radius: 0 20% 0 0;
```

Comment:
<img width="877" alt="2" src="https://github.com/medijdir/portfolio_project/assets/77843050/6fd606f8-cafd-4171-81f2-2a25cc2bf350">

- [ ] I think if you just use `border-radius` to 50% will make the deal, as for now, your 20% is not really what we want here
- [ ] Consider adding the arrow down icon in the language section as shown in the design, please